site/docs/skylark/performance.md

```diff
@@ -197,14 +197,163 @@ ctx.actions.run(
 
 ## Performance profiling
 
-To profile your code and analyze the performance, use the `--profile` flag:
+To profile your code and analyze the performance you have two options:
+
+- use the `--profile` flag and the `analyze-profile` command, or
+- use the new `--experimental_generate_json_trace_profile` flag and load the
+  resulting JSON profile in `chrome://tracing` (recommended).
+
+### --profile and analyze-profile
+
+This profiling method consists of two steps, first you have to execute your
+build/test with the `--profile` flag, e.g.
```

Comment:
Use "for example" instead of "e.g." as some non-native English speakers might not understand it.